### PR TITLE
Issue #10 by kumiko, naoi:  Allow the existing user to log in

### PR DESCRIPTION
**Pull request by kumikoono**
The previous patch doesn't work with multiple scenarios in a row.

### CODE REVIEW

File: src/Drupal/DrupalExtension/Context/DrupalContext.php [hunk 1]
```diff
@@ -631,9 +631,16 @@ private function getScreenshotParameter(string $name): ?string {
...
    */
-  public function iAmLoggedInAsUser($name): void {
-    $user = user_load_by_name($name);
-    $this->getSession()->visit(user_pass_reset_url($user) . '/login');
-  }
+  public function iAmLoggedInAsUser($name) {
+
+    //Another solution using user_pass_reset_url() is independent from drush,
+    // but it works only once.
+    $uli = $this->getDriver('drush')->drush('user:login', [
```
Comment (naoi) [inline src/Drupal/DrupalExtension/Context/DrupalContext.php]: @kumikoono What `$uli` stands for?  User log-in?  Or typo of `uri`?  Can you change the variable name which makes even more clear meaning?

**Review by naoi — CHANGES_REQUESTED** [review]
@kumikoono I posted my comment.  Thanks

File: src/Drupal/DrupalExtension/Context/DrupalContext.php (hunk 1 above)
Comment (kumikoono) [reply src/Drupal/DrupalExtension/Context/DrupalContext.php]: It meant "uli" as it aligns with the alias of "`drush user:login`",  `drush uli.`  
https://drushcommands.com/drush-9x/user/user:login/

Anyway, I renamed it "user_login"

File: src/Drupal/DrupalExtension/Context/DrupalContext.php [hunk 2]
```diff
@@ -631,9 +631,16 @@ private function getScreenshotParameter(string $name): ?string {
...
+    // but it works only once.
+    $base_url = $this->getMinkParameter('base_url');
+    $user_login= $this->getDriver('drush')->drush('user:login', [
+      "--name=" . $name,
+      "--no-browser",
+      "--uri=" . $base_url,
+    ]);
+    $this->getSession()->visit(trim($user_login));
   }
-
```
Comment (naoi) [inline src/Drupal/DrupalExtension/Context/DrupalContext.php:638]: @kumikoono I think we should keep this blank line since this is the end of this class.

**Comment by naoi** [comment]
@kumikoono Thank you for the update.  I merged the PR.  Thanks.

**Pull request merged**